lib/utils/place-outdial.js

```diff
@@ -358,6 +359,24 @@ class SingleDialer extends Emitter {
         this.logger.info(err, 'place-outdial:_configMsEndpoint - error enable inband DTMF');
       }
     }
+
+    const origDestroy = this.ep.destroy.bind(this.ep);
+    this.ep.destroy = async() => {
+      try {
+        if (this.dialTask.transcribeTask) {
```

Comment:
only do this if we have a nested transcribe **and** JAMBONES_DIAL_TRANSCRIBE_EP_DESTROY_DELAY_MS has been set

Reply:
fixed